### ISSUE TITLE
Speakers for 2018 meeting -Promote

### ISSUE THREAD

**Issue by DeniseColbert** [FreeUKGen/Coordination#791]
Made a new story as info urgently needed for ad copy for Sept issue of family tree magazine.

Deadline is 13th July

**Comment by DeniseColbert** [comment]
![2018 meeting](https://user-images.githubusercontent.com/19173257/42439788-b6d529d6-835b-11e8-9707-fdc9c2e5a891.PNG)

**Comment by PatReynolds** [comment]
Contacted again 9/7/2018

**Comment by DeniseColbert** [comment]
Ad copy sent 13/07

**Comment by DeniseColbert** [comment]
[FreeGenUKSepVersion2.pdf](https://github.com/FreeUKGen/Coordination/files/2192829/FreeGenUKSepVersion2.pdf)

**Comment by DeniseColbert** [comment]
Sent draft schedule to Pat 3/8

**Comment by DeniseColbert** [comment]
Draft program created, waiting on green light from speakers for bios:
https://drive.google.com/file/d/1J4ok2xYmNhfItabOgcKiuhw8edgAr_DM/view?usp=sharing
